generate/input/callbacks.json

```diff
@@ -424,7 +424,7 @@
       },
       {
         "name": "stash_id",
-        "cType": "const int *"
+        "cType": "const git_oid *"
```

Comment:
Don't trust the docs :)

On Thu, Mar 19, 2015, 6:18 AM Tim Branyen <notifications@github.com> wrote:

> In generate/input/callbacks.json
> <https://github.com/nodegit/nodegit/pull/495#discussion_r26752085>:
>
> > @@ -424,7 +424,7 @@
> >        },
> >        {
> >          "name": "stash_id",
> > -        "cType": "const int *"
> > +        "cType": "const git_oid *"
>
> omg....
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodegit/nodegit/pull/495/files#r26752085>.
>